src/EndpointManager.sol

```diff
@@ -97,9 +99,10 @@ abstract contract EndpointManager is
         _rateLimitDuration = rateLimitDuration;
     }
 
-    function initialize() public initializer {
+    function __EndpointManager_init() internal onlyInitializing {
         // TODO: shouldn't be msg.sender but a separate (contract) address that's passed in the initializer
         __Ownable_init(msg.sender);
+        // TODO: check if it's safe to not initialise reentrancy guard
```

Comment:
this is more of a note to self (we should definitely initialise it, I was just curious if behaves sanely without initialisation)